services/harmony/test/service-image-tags.ts

```diff
@@ -34,6 +34,10 @@ const serviceImages = {
   'podaac-l2-subsetter': 'sit',
   'query-cmr': 'latest',
   'sds-maskfill': 'latest',
+  'giovanni-adapter': 'latest',
```

Comment:
TSError: ⨯ Unable to compile TypeScript:
Error: test/service-image-tags.ts(37,3): error TS1117: An object literal cannot have multiple properties with the same name.
Error: test/service-image-tags.ts(38,3): error TS1117: An object literal cannot have multiple properties with the same name.
Error: test/service-image-tags.ts(40,3): error TS1117: An object literal cannot have multiple properties with the same name.

geoloco, batchee, and giovanni-adapter are already defined - looks like they were copied twice

Reply:
ah, must've been a bad merge. Removed the duplicates.

@chris-durbin also renamed to `subset_band_name` where applicable.